<launch>

  <param name="use_sim_time" value="false" />

  <!-- 打开地图 -->
  <node name="map_server" pkg="map_server" type="map_server" args="$(find bitathome_navigation_control)/maps/map.yaml" />

  <!-- 打开激光定位 提供码盘数据>
  <node pkg="laser_scan_matcher" type="laser_scan_matcher_node" name="laser_scan_matcher_node" output="screen">
    <param name="fixed_frame" value="odom" />
  </node -->
  <node pkg="hector_mapping" type="hector_mapping" name="hector_mapping" output="screen" >
    <param name="pub_map_odom_transform" value="true" />
    <param name="map_frame" value="map_map" />
    <param name="base_frame" value="base_link" />
    <param name="odom_frame" value="base_link" />
    <param name="laser_frame" value="laser" />
  </node>

  <!-- 打开底座控制 输出串口打开关闭信息 -->
  <node pkg="bitathome_hardware_control" name="hc_motor_cmd" type="hc_motor_cmd.py" output="screen" />
  
  <node pkg="bitathome_navigation_control" name="my_move_base" type="my_move_base_navigation.py" output="screen" />
  <!-- node pkg="bitathome_navigation_control" name="my_map_show" type="mapshow.py" output="screen" / -->
  <node pkg="joy" name="joy_node" type="joy_node" />
  <node pkg="bitathome_navigation_control" name="joy_tf_control" type="joy_tf_control.py" />
  <node pkg="bitathome_navigation_control" name="my_map" type="my_get_path" output="screen" />

  <!-- node pkg="tf" type="static_transform_publisher" name="map_odom_broadcaster" args="0 0 0 0 0 0 /map /odom 100" / -->

</launch>
